tests/unit/Commands/SyncWoocommerceProductsTest.php

```diff
@@ -121,6 +126,79 @@ public function testSimpleProductImport()
         $this->assertAttributeOptionOrder();
     }
 
+    public function testProductImageImport()
+    {
+        $storekeeperProductId = 20;
+        $imageCdnPrefix = 'test';
+        // Set CDN to false first
+        StoreKeeperOptions::set(StoreKeeperOptions::IMAGE_CDN, 'no');
+        StoreKeeperOptions::set(StoreKeeperOptions::IMAGE_CDN_PREFIX, $imageCdnPrefix);
+        $this->initializeTest($storekeeperProductId);
+
+        $originalProductData = $this->getReturnData(self::DATADUMP_IMAGE_PRODUCT_FILE);
+
+        // Get the simple products from the data dump
+        $originalProductData = $this->getProductsByTypeFromDataDump(
+            $originalProductData,
+            self::SK_TYPE_SIMPLE
+        );
+
+        // Test if image is downloaded
+        $this->assertDownloadedImage($originalProductData);
+
+        // Set CDN to true
+        StoreKeeperOptions::set(StoreKeeperOptions::IMAGE_CDN, 'yes');
+        $syncCommand = new SyncWoocommerceSingleProduct();
+        $syncCommand->runSync([
+            'storekeeper_id' => $storekeeperProductId,
+        ]);
+
+        foreach ($originalProductData as $productData) {
+            $original = new Dot($productData);
+
+            // Retrieve product(s) with the storekeeper_id from the source data
+            $wcProducts = wc_get_products(
+                [
+                    'post_type' => 'product',
+                    'meta_key' => 'storekeeper_id',
+                    'meta_value' => $original->get('id'),
+                ]
+            );
+
+            // Get the simple product with the storekeeper_id
+            /* @var \WC_Product $wcSimpleProduct */
+            $wcSimpleProduct = $wcProducts[0];
```

Comment:
maybe make a function from it `assertCdnImage`

Reply:
Added assert function.